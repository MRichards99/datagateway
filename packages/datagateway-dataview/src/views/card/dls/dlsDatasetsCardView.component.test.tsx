import { createShallow } from '@material-ui/core/test-utils';

describe('DLS Datasets - Card View', () => {
  let shallow;

  beforeEach(() => {
    shallow = createShallow();
  });

  it.todo('renders correctly');
});
