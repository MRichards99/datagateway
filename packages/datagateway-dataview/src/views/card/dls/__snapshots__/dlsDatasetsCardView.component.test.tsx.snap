// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DLS Datasets - Card View renders correctly 1`] = `
<ContextProvider
  value={null}
>
  <DLSDatasetsCardView
    addToCart={[Function]}
    cartItems={Array []}
    data={
      Array [
        Object {
          "CREATE_TIME": "2019-07-23",
          "ID": 1,
          "INVESTIGATION_ID": 1,
          "MOD_TIME": "2019-07-23",
          "NAME": "Test 1",
        },
      ]
    }
    fetchCount={[Function]}
    fetchData={[Function]}
    fetchDetails={[Function]}
    fetchSize={[Function]}
    fetchTypeFilter={[Function]}
    filterData={Object {}}
    investigationId="1"
    loadedCount={true}
    loadedData={true}
    proposalName="test"
    pushFilters={[Function]}
    pushPage={[Function]}
    pushQuery={[Function]}
    query={
      Object {
        "filters": Object {},
        "page": null,
        "results": null,
        "search": null,
        "sort": Object {},
        "view": null,
      }
    }
    removeFromCart={[Function]}
    store={
      Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      }
    }
    totalDataCount={1}
  />
</ContextProvider>
`;
